src/forklift/cli.py

```diff
@@ -139,11 +139,14 @@ def start_lift(file_path=None):
 
     lift.process_crates_for(pallets, core.update)
 
+    lift.process_pallets(pallets)
+
```

Comment:
Are you thinking showing the total elapsed time after each of the process and copy calls? What about showing the total time that each of them takes? Something like:
```
process_crates time: 10 min
process_pallets time: 5 min
copy_data time: 7min
```

or something else?